addons/mail/static/src/models/call_settings_menu.js

```diff
@@ -3,11 +3,12 @@
 import { browser } from "@web/core/browser/browser";
 
 import { registerModel } from '@mail/model/model_core';
-import { attr, one } from '@mail/model/model_field';
+import { clear, replace } from '@mail/model/model_field_command';
+import { one } from '@mail/model/model_field';
 
 registerModel({
     name: 'CallSettingsMenu',
-    identifyingFields: ['userSetting'],
+    identifyingFields: [['chatWindowOwner', 'threadViewOwner']],
```

Comment:
don't forget to rebase with new changes with identifying fields (here it becomes `identifyingMode: 'xor'` and each field should have `identifying: true`